recipes/glslang/all/conanfile.py

```diff
@@ -169,7 +173,6 @@ def package_info(self):
             self.cpp_info.components["glslang-core"].requires.append("oglcompiler")
         if self.options.hlsl:
             self.cpp_info.components["glslang-core"].defines.append("ENABLE_HLSL")
-            self.cpp_info.components["glslang-core"].requires.append("hlsl")
```

Comment:
Shouldn't this ``self.options.hlsl`` condition be ``if has_hlsl`` too?

Reply:
Yes, good spot.